### ISSUE TITLE
Tasks to do for bbgdm package. 

### ISSUE THREAD

**Issue by skiptoniam** [skiptoniam/bbgdm#1]
- [x] Just catch bb parameter estimates to reduce model size.
- [ ] Fix link function in predict.bbgdm
- [ ] Fix i-spline transformation in predict.bbgdm
- [ ] Add return eta to predict.bbgm
- [ ] Add return whole dissimilarity to predict.bbgdm
- [ ] Add a function to incorporate custom dissimilarity
- [ ] Extrapolation beyond data?

**Comment by Leitemfa** [comment]
Is there any interest in including other family functions such as negative binomial?

**Comment by skiptoniam** [comment]
No. Because we are modelling dissimilarity data, so only Bernoulli, binomial or beta are appropriate.

**Comment by Leitemfa** [comment]
Hi @skiptoniam , thanks for the quick reply. I was just wondering if there will be a away to analyze dissimilarity from abundance instead only presence absence as it is available now.

**Comment by Leitemfa** [comment]
Is there any plan to include that?

**Comment by skiptoniam** [comment]
@Leitemfa usually modelling dissimilarities from abundance data is done via some specific type of dissimilarity metric like bray-curtis (and others). If you want to model multiple species where you have abundance data you might want to consider multiple species models like [mvabund](https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/j.2041-210X.2012.00190.x), joint species distribution models [HMSC](https://github.com/guiblanchet/HMSC) or our other work which models multiple species as groups [ecomix](https://github.com/skiptoniam/ecomix). Under all these models you can deal with multiple species and abundance data explicitly, it just really depends on what your question is...

**Comment by Leitemfa** [comment]
Hi @skiptoniam , yes, I understand that, but I see in the bbgdm package that it only analysis presence absence data, but I would like to use it to investigate abundance data, just like in gdm package but including the boostrap framework available in bbgdm. Is that possible?

**Comment by skiptoniam** [comment]
@Leitemfa yep If you chose the "bray_curtis" option in dissimilarity.

**Comment by Leitemfa** [comment]
Well, but when I add an abundance data and change the dism_metric to "bray_curtis" I got the message:

`Error in pam2dissim(sp.dat, dism_metric) : 
  bbgdm only works with presence-absence - transform your abundances to binary data`

**Comment by skiptoniam** [comment]
Woops. Ok, I can change that. I'll let you know when I have.

**Comment by Leitemfa** [comment]
Thank you very much @skiptoniam . Looking foward to see that.

Kind regards,
Marcio
